lib/core/utils/preload-cssom.js

```diff
@@ -75,14 +149,24 @@ function loadCssom({ root, shadowId }, timeout, convertTextToStylesheetFn) {
 	});
 
 	// iterate to decipher multi-level nested sheets if any (this is essential to retrieve styles from shadowDOM)
-	sheets.forEach(sheet => {
+	sheets.forEach((sheet, sheetIndex) => {
+		/* eslint max-statements: ["error", 20] */
+
+		// padding sheet index with root index to create priority
+		const sheetLevelPriority = Number(`${rootIndex}.${sheetIndex}`);
```

Comment:
Doesn't this break if we've got more than 10 sheets? Is this priority something you came up with or is it part of a spec some place? This seems fragile.